s3/src/main/scala/akka/stream/alpakka/s3/S3Headers.scala

```diff
@@ -118,6 +120,12 @@ final class S3Headers private (val cannedAcl: Option[CannedAcl] = None,
  * Convenience apply methods for creation of canned acl, meta, storage class or custom headers.
  */
 object S3Headers {
+
+  /**
+   * Empty set of headers
+   */
+  val empty: S3Headers = S3Headers()
+
   def apply() = new S3Headers()
```

Comment:
I'd suggest you do it like: 
```
val empty = new S3Headers()
def apply() = empty
def create() = empty
```

Reply:
Would you prefer keeping the `empty` on the API too as I did or just hide it and use in `apply` and `create`?